src/ssl_load.c

```diff
@@ -5102,6 +5102,22 @@ int wolfSSL_CTX_use_RSAPrivateKey(WOLFSSL_CTX* ctx, WOLFSSL_RSA* rsa)
 
 #ifdef OPENSSL_EXTRA
 
+static char* wolfSSL_getenv_memcpy(const char* varName) {
```

Comment:
Is there a better place for this function?

Reply:
You could wrap the call to XGETENV with a duplicate string call.
That is XSTRDUP but using XMALLOC. Which I guess can go into wc_port.c???